create_aruco_dict.py

```diff
@@ -0,0 +1,25 @@
+import sys
+sys.path.append('/usr/local/lib/python2.7/site-packages')
+import cv2
+import numpy as np
+import cv2.aruco as aruco
+
+#small_dict = aruco.generateCustomDictionary(6, 3)
```

Comment:
Is this dead code?

Reply:
yes. I forgot this file would be included and hadn't tidied it up.